import logging

logging.exception("Hello world")